pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -150,6 +214,24 @@ def conv_block(
 
 
 class JoinBlock(nn.Module):
+    r"""JoinBlock from :cite:`ULVL2016`.
+
+    This block concatenate two inputs along the ``channel _dim`` with previous
```

Comment:
1. In cats an arbitrary number of inputs.
2. 
```suggestion
    This block concatenate two inputs along the ``channel _dim`` with prefixed
```